### PR TITLE
Adding style guide

### PR DESCRIPTION
**Pull request by gvwilson**
Restored from <https://github.com/datacamp/teach-documentation/blob/master/style_guide.md> at @richierocks suggestion.

### CODE REVIEW

**Comment by gvwilson** [comment]
Thanks @richierocks - @ramnathv can you please check and merge?

**Comment by ncarchedi** [comment]
Is this ready to be merged? Seems @yashasroy should be reviewer on this, not Ramnath.

File: docs/challenges/guides/style.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Challenge Style
+
+Always presents the original DataFrame in the context/assignment of
```
Comment (yashasroy) [inline docs/challenges/guides/style.md]: Typo: Always presents --> Always present

Comment (gvwilson) [reply docs/challenges/guides/style.md]: fixed in c6c37a6d545

**Pull request merged**